apps/src/gamelab/blocks.js

```diff
@@ -257,10 +257,13 @@ export default {
     };
 
     generator.gamelab_behavior_get = function () {
-      return [
-        Blockly.JavaScript.variableDB_.getName(
+      const name = Blockly.JavaScript.variableDB_.getName(
             this.getTitleValue('VAR'),
-            Blockly.Procedures.NAME_TYPE),
+            Blockly.Procedures.NAME_TYPE);
+      // TODO: add support for passing extra params into this block
```

Comment:
TODO

Reply:
Have you had started working on this already? It's closely related to your behavior editor changes.

Reply:
Are these the extra params beyond the `this sprite` param?  I only mutate the registered call blocks, via [`updateCallerParams_`](https://github.com/code-dot-org/blockly/blob/868084e83cc33d24b992039f40a6b04728aeb7d4/blocks/procedures.js#L180-L187).

Does this block implement [`setProcedureParameters`](https://github.com/code-dot-org/blockly/blob/04705fd7494610e1f6a2b682b3f33ebbc3fb3161/core/utils/procedures.js#L294-L299)?

Reply:
Correct, it's the extra params beyond `this sprite`. It does not implement `setProcedureParameters` yet.